pro/src/screens/OfferIndividual/StocksThing/StocksThing.tsx

```diff
@@ -280,7 +282,7 @@ const StocksThing = ({ offer }: IStocksThingProps): JSX.Element => {
 
   const cannotDeleteStock = isDisabled || isSynchronized
 
-  actions.push({
+  actions.unshift({
```

Comment:
(détail) 
Si on toujours le bouton supprimer le stock tu peux directement
```suggestion
let actions: IStockFormRowAction[] = [{callback:
      /* istanbul ignore next: DEBT, TO FIX */
      formik.values.bookingsQuantity !== '0'
        ? deleteConfirmShow
        : onConfirmDeleteStock,
    label: 'Supprimer le stock',
    disabled: cannotDeleteStock,
    Icon: IcoTrashFilled,
  }]
```